core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -75,7 +75,9 @@ object ApiVersion {
     // Several request versions were bumped due to KIP-219 (Improve quota communication)
     KAFKA_2_0_IV1,
     // Introduced new schemas for group offset (v2) and group metadata (v2) (KIP-211)
-    KAFKA_2_1_IV0
+    KAFKA_2_1_IV0,
+    // New Fetch, OffsetsForLeaderEpoch, and ListOffsets schemas (KIP-320)
```

Comment:
nits: can we include Metadata, OffsetCommit, TxnOffsetCommit and OffsetFetch in the comment for completeness?

Reply:
I only mentioned the inter-broker protocols that were affected since we don't bump this version for client-only APIs. Does that make sense?

Reply:
Yeah this makes sense.